### DISCUSSION TITLE
Feature request: Immediate warning sound when any of the doors are open

### DISCUSSION THREAD

**Discussion by mabkenar** [rgerganov/gen5fw#10] [Ideas]
The warning should **not** come only after a minimum speed is reached, rather it should come immediately above 0 km/h, i.e. when the wheels start to roll. Alternatively, the warning may start immediately "on Shift" (when a gear other than P or N is selected).

There is a related sub-menu in the settings called "Auto lock" with two options, namely "Enable [auto lock] on speed" (default choice) and "Enable [auto lock] on shift". I tried to set this setting to "on shift" to see if it has the effect that I am looking for. After doing so, while I could hear that all the //closed// doors were locked immediately after I put the car into D or R, when I "forgot" to close one of the doors, the door is simply neglected and no warning sound is played at all!

![image](https://user-images.githubusercontent.com/3006332/217352536-c8a88742-29e1-4e9e-9d93-e3245b64cfe2.png)
![image](https://user-images.githubusercontent.com/3006332/217352572-ef61bcbf-47b4-4425-9bf3-8e1f7c6a930c.png)
![image](https://user-images.githubusercontent.com/3006332/217352600-48f06984-b606-453d-a133-eb3727d50339.png)

Please feel free to add your thoughts, suggestions, refinement, and alternatives related to this feature request.

**Comment by rgerganov** [comment]
This sounds reasonable but I am not sure if this functionality is part of the head unit. My understanding is that the head unit is only for infotainment and navigation. Not sure if it is connected to this particular screen.
